uber/config.py

```diff
@@ -120,6 +120,18 @@ def PAGE_PATH(self):
     def PAGE(self):
         return cherrypy.request.path_info.split('/')[-1]
 
+    @property
+    def SUPPORTER_COUNT(self):
+        with sa.Session() as session:
+            attendees = session.query(sa.Attendee)
+            individual_supporters = attendees.filter(or_(sa.Attendee.paid == self.HAS_PAID,
+                                                         sa.Attendee.paid == self.REFUNDED),
+                                                         sa.Attendee.amount_extra == c.SUPPORTER_LEVEL).count()
+            group_supporters = attendees.join(sa.Attendee.group).filter(sa.Attendee.paid == self.PAID_BY_GROUP,
+                                                                        sa.Attendee.amount_extra == c.SUPPORTER_LEVEL,
+                                                                        sa.Group.amount_paid > 0).count()
```

Comment:
You probably want to just check ``sa.Attendee.amount_paid``.  The way we track group supporters is that the attendee themselves is ``PAID_BY_GROUP`` but still has an ``amount_extra`` (which you're checking) and an ``amount_paid``.  The way you're doing it, you might end up counting Supporters who belong to groups where the group has paid but the Supporter didn't pay for their kicked-in-extra amounts.